user_interaction/templates/user_interaction/home_users.html

```diff
@@ -0,0 +1,118 @@
+
```

Comment:
I like the overview, but right now the Activities page has pretty much the same functionality (with the only difference being the 7 days vs 14 days "preview")